VajkVarga/Library/Library/BookManager.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Library.Entities;
+
+namespace Library
+{
+    internal class BookManager
+    {
+        private readonly List<Book> Books;
+
+        public BookManager(List<Book> books)
+        {
+            Books = books.Distinct().ToList();
```

Comment:
HashSet is lehetne éppen, akkor a Distinct és a ToList megspórolható, illetve későbbi bővítésnél is Distinct maradna.